users/assets/js/script.js

```diff
@@ -0,0 +1,303 @@
+            reader.addEventListener("load", function () {
+                imgPreview.src = reader.result;
+                imgPreviewWrapper.style.display = "block";
+            });
+            reader.readAsDataURL(file);
+            imgFileName.innerText = file.name;
+            imgFileNameError.innerText = "";
+        } else {
+            imgPreview.src = "#";
+            imgPreviewWrapper.style.display = "none";
+        }
+    });
+    }
+     //image visibility
+     if(document.body.contains(removeImagePreviewIcon)){
+        removeImagePreviewIcon.addEventListener('click', function() {
+            removeImagePreview(fileInput,imgPreviewWrapper,imgPreview,imgFileName,imgFileNameError);
+        }); 
+     }
+});
+
+function validatePassword(password) {
+    var passwordPattern = /^(?=.*[A-Z])(?=.*\d)(?=.*[!@#$%()]).{6,}$/;
+
+    // Password validation
+    if (!passwordPattern.test(password)) {
+        document.querySelector("#password_error").innerText = "Password does not match criteria";
+        return false;
+    } else {
+        document.querySelector("#password_error").innerText = "";
+        return true;
+    }
+}
+
+function confirmPassword(password, confirm_password) {
+    // Password confirmation validation
+    if (password !== confirm_password) {
+        document.querySelector("#confirm_password_error").innerText = "Passwords do not match.";
+        return false;
+    } else {
+        document.querySelector("#confirm_password_error").innerText = "";
+        return true;
+    }
+}
+
+function togglePasswordVisibility(passwordInput,passwordVisibility) {
+    
+    if (passwordInput.type === "password") {
+        passwordInput.type = "text";
+        passwordVisibility.classList.add("fa-eye-slash");
+        passwordVisibility.classList.remove("fa-eye");
+    } else {
+        passwordInput.type = "password";
+        passwordVisibility.classList.add("fa-eye");
+        passwordVisibility.classList.remove("fa-eye-slash");
+    }
+}
+
+function validateEmail(email) {
+    var emailValid = /^[a-zA-Z0-9._%+-]+@[a-zA-Z0-9.-]+\.[a-zA-Z]{2,}$/;
+
+    if (!emailValid.test(email)) {
+        document.querySelector("#email_error").innerText = "Email ID is not valid";
+        return false;
+    } else {
+        document.querySelector("#email_error").innerText = "";
+    }
+    return true;
+}
+
+function removeChars(phoneInput){
+    phoneInput.value = phoneInput.value.replace(/[^0-9]/g, '');
+}
+
+
+function validateForm() {
+    var username = document.getElementById("username").value;
+    var name = document.getElementById("name").value;
+    var email = document.getElementById("email").value;
+    var password = document.getElementById("password").value;
+    var confirm_password = document.getElementById("confirm_password").value;
+
+    if (username === "" || name === "" || email === "") {        
+        document.querySelectorAll("#username_error,#name_error,#email_error").forEach(function(element) {
```

Comment:
use a common class for error message instead of using individual ID's

Reply:
This is fixed